extensions/types/vscode-mssql.d.ts

```diff
@@ -721,6 +722,14 @@ declare module 'vscode-mssql' {
 		defaultDeploymentOptions: DeploymentOptions;
 	}
 
+	export interface SavePublishProfileParams {
```

Comment:
I'd suggest only updating this file around the same time vscode-mssql is also updated with this, so that the files stay in sync